arches/app/views/search.py

```diff
@@ -348,28 +367,17 @@ def get_dsl_from_search_string(request):
 
 
 def search_results(request, returnDsl=False):
```

Comment:
we should try and push as much of this code into `base.py` or `core_search.py`.  The search view should just be a light weight wrapper that calls out to the core search component to perform much of the core logic.